.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: Testing Interoperability
```

Comment:
we should avoid to put this file here since it will be done in a different branch, right?

Reply:
okey, i'll delete it from the branch